vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/GeneratedVaadinTextField.java

```diff
@@ -498,21 +497,6 @@ protected String getLabelString() {
         return getElement().getProperty("label");
     }
 
-    /**
-     * <p>
-     * Description copied from corresponding location in WebComponent:
-     * </p>
-     * <p>
-     * String used for the label element.
-     * </p>
-     *
-     * @param label
-     *            the String value to set
-     */
-    protected void setLabel(String label) {
-        getElement().setProperty("label", label == null ? "" : label);
-    }
```

Comment:
This method was not removed in the original PR. Since it's a public class that might be referenced in existing applications, let's restore this method.